lib/distributed_simulator.ex

```diff
@@ -52,6 +52,13 @@ defmodule DistributedSimulator do
     {cells, neighbors}
   end
 
+  def populate grid, [] do
```

Comment:
Is it function for adding Mocks to the cells? Why do we need two versions of the function?

Reply:
yes. and yes, populate should accept grid parameter in both cases.